src/python/foglamp/model/config.py

```diff
@@ -0,0 +1,56 @@
+"""
+Environment variables
```

Comment:
same as src/python/foglamp/config.py, here we can say "reads database config and prepares the db connection string"